{
  "id": "uno-ooo-parser",
  "version": "0.1.8",
  "timestamp": "2021-12-28 10:14:21+00:00",
  "name": "ConnectException",
  "type": "exception",
  "namespace": "com.sun.star.io",
  "parser_args": {
    "sort": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "from_imports": [
      [
        ".socket_exception",
        "SocketException"
      ]
    ],
    "from_imports_typing": [],
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "name": "ConnectException",
    "namespace": "com.sun.star.io",
    "imports": [],
    "extends": [
      "com.sun.star.io.SocketException"
    ],
    "desc": [
      "Signals that an error occurred while attempting to connect a socket to a remote address and port.",
      "",
      "Typically, the connection was refused remotely (e.g., no process is listening on the remote address/port)."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1io_1_1ConnectException.html",
    "items": {}
  }
}